content/docs/behaviors/contribute.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Learning to Contribute
+weight: 12
+description: At Axelerant, engineers are encouraged and supported to have a passion for learning new things every day. With technology constantly evolving, there is always something new to discover and explore. Being enthusiastic about learning not only helps you stay current with the latest trends and advancements, but it also keeps you engaged and motivated in your work.
+---
+
+# Learning to Contribute
```

Comment:
May be my intention with this was to talk about taking the initiative to help when colleagues face challenges, not only knowing the solution but also taking the initiative to learn about or research the challenges and come up with solutions.

Reply:
Fair enough, but there still is significant overlap. The risk with having overlapping content is that it can get very confusing over time. That is why I'm asking you to see what is new in this page and add it over there. Feel free to make edits to that page to have it flow better.

Reply:
@hussainweb I am preparing another separate file for this. I am going to create a separate page about open source contribution based on the [document](https://docs.google.com/document/d/1icMEor7RiEgdr_Rb2St9_TkOqLwigsoK7D7mVI1yh54/edit) prepared by Gaurav Mahlawat. And removing the contrib file. Please review and merge (if good enough) the other page in this PR.
Thanks.